### PR TITLE
CASMTRIAGE-5937 Update sls.md with Authenication Token details.

### PR DESCRIPTION
**Pull request by ganeshs-hpe**
# Description

# Checklist

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by ganeshs-hpe** [comment]
Update sls.md with Authenication Token details

File: api/sls.md [hunk 1]
```diff
@@ -141,6 +143,7 @@ Accept: application/json
 ```
 
 ```shell
+
```
Comment (rustydb) [inline api/sls.md]:
```suggestion
```

File: api/sls.md [hunk 2]
```diff
@@ -129,6 +129,8 @@ Base URLs:
 
 Endpoints which request information about hardware
 
+Refer link [Retrieve an Authentication Token](Retrieve_an_Authentication_Token.md) to get Authentication Token before using the code samples.
```
Comment (gbaker-hpe) [inline api/sls.md]:
```suggestion
Refer to the ['Retrieve an Authentication Token'](Retrieve_an_Authentication_Token.md) documentation for information regarding retrieving an authentication token.
```

**Pull request merged**